database.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const log = require('./log.js');
+
+module.exports = {
```

Comment:
Since I see that you're using babel now it might be worth switching to the ES6 module syntax instead.